app/modules/projects/models.py

```diff
@@ -35,6 +47,8 @@ class Project(db.Model, Timestamp):
         'ProjectUserMembershipEnrollment', back_populates='project'
     )
 
+    encounter_members = db.relationship('ProjectEncounter', back_populates='project')
```

Comment:
Not at all certain about this name, encounters are not members of a project as such but no better name sprang to mind

Reply:
yeah, encounters are (really: "can be") _part of_ or _included in_ projects, i suppose.  "member" kinda makes sense -- to coders?  :)  i cant come up with any more clever name anyway.

Reply:
Records? Can it just be 'encounters'?

Reply:
@colinwkingen, I like records, that works. I'll do that.
 No it can't be encounters as that is the name of the property function line 64 which matches that for records at line 61